vscode-dotnet-runtime-library/src/Acquisition/AcquisitionInvoker.ts

```diff
@@ -101,8 +101,7 @@ You will need to restart VS Code after these changes. If PowerShell is still not
     private escapeFilePath(path: string): string {
         if (os.platform() === 'win32') {
             // Need to escape apostrophes with two apostrophes
-            const dotnetInstallDirEscaped = path.replace(/'/g, `''`);
-
+            const dotnetInstallDirEscaped = path.replace(/'/g, `\'`);

```

Comment:
This *might* work but it depends on how it is called. I'd be interested to see a unit test that resolves the command call (with path to the install script) and checks to see it works properly. Not sure what unit testing is available in this repo.

Reply:
Thanks for the comment! A unit test is added with the changes we discussed